pkg/clustermesh/common/config.go

```diff
@@ -177,3 +177,22 @@ func (cdw *configDirectoryWatcher) close() {
 	close(cdw.stop)
 	cdw.watcher.Close()
 }
+
+// ConfigFiles returns the list of configuration files in the given path. It
+// shall be used by CLI tools only, as it doesn't handle subsequent updates.
+func ConfigFiles(cfgdir string) (configs map[string]string, err error) {
+	files, err := os.ReadDir(cfgdir)
+	if err != nil {
+		return nil, err
+	}
+
+	configs = make(map[string]string)
+	for _, f := range files {
+		cfgfile := path.Join(cfgdir, f.Name())
```

Comment:
I believe this should be `filepath.Join` instead.

Reply:
Ah, good point, thanks. I didn't know about that difference. I've updated the other two occurrences in the same file as well.